include/ibamr/IIMethod.h

```diff
@@ -168,6 +168,22 @@ class IIMethod : public IBStrategy
         void* ctx;
     };
 
+/*!
+     * Returns either the current configuration (at possibly different times in the timestep)
+     * or the reference configuration with a numeric vector
```

Comment:
This comment is not very clear. What does it mean "at possibly different times in the timestep"?
```suggestion
     * Returns either the current configuration or the reference configuration of the interface at the specified time.
```